models/org.go

```diff
@@ -233,7 +233,8 @@ func DeleteOrganization(org *User) (err error) {
 	}
 
 	if err = deleteUser(sess, org); err != nil {
-		return fmt.Errorf("deleteUser: %v", err)
+		// Note: don't wrapper error here.
```

Comment:
For me it's obvious to not wrapping this error, so IMHO you can drop this comment.